extension/package.json

```diff
@@ -396,6 +401,11 @@
         "command": "dvc.selectFocusedProjects",
         "category": "DVC"
       },
+      {
+        "title": "Set Self-Hosted Studio URL",
```

Comment:
Decided on just using one command instead of `Update` and `Add` commands.

Reply:
Ohhhh, ok. Well, if we want all commands to be consistent, `Add` makes more sense to me in other commands contexts. I'll go ahead and just use `Update` and `Add` commands 😄 for this too.